doajtest/testbook/dashboard/historical_stats.yml

```diff
@@ -0,0 +1,97 @@
+      - step: Log in with provided editor_0 credentials.
+        results:
+          - User dashboard is displayed.
+      - step: Verify the motivational banner at the top of the dashboard (green) and check number of finished applications.
+        results:
+          - The motivational banner is GREEN and shows the correct sum of applications finished in both groups.
+      - step: Verify historical statistics in the Activity section at the bottom.
+        results:
+          - Two groups are displayed.
+          - Only associate editors are shown in the Activity section for each group.
+          - Each group has the correct statistics for the current year.
+          - The number of READY/COMPLETED applications matches the "additional information" table.
+      - step: Log out.
+      - step: (Optional) Click the link at the bottom of the testdrive page to free resources after finishing this test session.
+  - title: Historical Statistics Display - Editor with 1 group
+    context:
+      role: editor_1
+      testdrive: statistics
+    steps:
+      - step: Navigate to "/testdrive/statistics" and wait for the page to load slowly. This should be done once at the beginning of the test session and not refreshed between individual tests.
+      - step: Log in with provided editor_1 credentials.
+        results:
+          - User dashboard is displayed.
+      - step: Verify the motivational banner at the top of the dashboard (green) and check number of finished applications.
+        results:
+          - The motivational banner is GREEN and shows the correct sum of applications finished in the group.
+      - step: Verify historical statistics in the Activity section at the bottom.
+        results:
+          - One group is displayed.
+          - Only associate editors are shown in the Activity section for the group.
+          - The group has the correct statistics for the current year.
+          - The number of READY/COMPLETED applications matches the "additional information" table.
+      - step: Log out.
+      - step: (Optional) Click the link at the bottom of the testdrive page to free resources after finishing this test session.
+  - title: Historical Statistics Display - Associate Editors with 0 finished applications
+    context:
+      role: associate editor
+      testdrive: statistics
+    steps:
+      - step: Navigate to "/testdrive/statistics" and wait for the page to load slowly. This should be done once at the beginning of the test session and not refreshed between individual tests.
+      - step: Log in with provided associate_editor_0 credentials.
+        results:
+          - User dashboard is displayed.
+      - step: Verify the motivational banner at the top of the dashboard (grey).
+        results:
+          - The motivational banner is GREY with appropriate information about no finished applications.
```

Comment:
I can't tell the difference between the green and the grey banners, is it subtle?

Reply:
Grey:
![image](https://github.com/user-attachments/assets/3c676134-a2e8-41d2-9131-54faca2ed7c0)

Green:
![image](https://github.com/user-attachments/assets/c41fc5ec-a69d-41f6-8613-29b04ac92599)